packages/flutter_tools/lib/src/commands/build.dart

```diff
@@ -67,7 +67,7 @@ abstract class BuildSubCommand extends FlutterCommand {
       globals.printStatus('💪 Building with sound null safety 💪', emphasis: true);
     } else {
       globals.printStatus(
-        'Building with unsound null safety',
+        'Building without sound null safety',
```

Comment:
While I think this is better, I'm not sure as a user what I'm supposed to do about this.

Flutter's style guide says log messages [must be actionable by the user](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#only-log-actionable-messages-to-the-console). Can this log be improved such that it alerts the user that they're mixing null-safe and non-null-safe packages, and they therefore do not get sound null safety?

If not, can we just ditch the log?

Reply:
As a user you are supposed to be reminded that you are in an incomplete migration state, and take steps to move towards completing it. The origin of this was https://github.com/dart-lang/sdk/issues/44234, which was exactly about a case like that.